template-manager-cmtemplate/src/main/resources/definitions/cm-cloud-storage-location-specification.json

```diff
@@ -98,6 +98,28 @@
       "relatedMissingServices": ["NAMENODE"],
       "requiredForAttachedCluster": true,
       "supportedStorages": ["ADLS", "WASB", "S3", "GCS", "ADLS_GEN_2"]
+    },
+    {
+      "type": "RANGER_DATA",
+      "propertyName": "cloud_data_location_url",
+      "propertyFile": "ranger-audit",
+      "description": "This is the path where Ranger stores its data",
+      "defaultPath": "{{{ defaultPath }}}/ranger/data",
+      "propertyDisplayName": "Ranger Cloud Data Location URL",
```

Comment:
Should we display something related to Raz here?

Reply:
Not technically Raz stuff.  This is all on Ranger.